app/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+// strapi service devcontainer
```

Comment:
I think you have the changes from https://github.com/clicepfl/clic-website-v2/pull/38 here. Have you checked out from 37-add-nextjs-app-to-devcontainers instead of the current main? Consider rebasing so that we have a distinct set of changes 👍